src/OrchardCore.Modules/OrchardCore.Navigation/NavigationShapes.cs

```diff
@@ -39,33 +39,38 @@ public void Discover(ShapeTableBuilder builder)
                     var viewContextAccessor = context.ServiceProvider.GetRequiredService<ViewContextAccessor>();
                     var viewContext = viewContextAccessor.ViewContext;
 
-                    var navigationManager = context.ServiceProvider.GetRequiredService<INavigationManager>();
+                    var navigationManagers = context.ServiceProvider.GetServices<INavigationManager>();
```

Comment:
Here is where I was not able to get the registered services when using the ServiceCollectionExtensions.cs for registering the admin menu INavigationManager.